File: code/12095.py
s = input()
n = len(s)
dmax = 0
d= 0
for c in range(n):
  if s[c] != " ":
    d = d + 1 
    if d > dmax:
      dmax = d
      v = c - d + 1
  else:
    d = 0
p = s[v:v+dmax]
print(p)

File: code/12095.cpp
#include <iostream>
#include <string>
using namespace std;
int main() 
{
    string s;
    getline(cin,s);
    int dmax=0, d=0;
    for (int i=0;i<s.length(); i++) {
        if (s[i]==' ') {
            d=0;
        }
        else {
            d++;
            if (d>dmax) dmax=d;
        }
    }
    cout<<dmax;
    return 0;
}
